rasa_core/training/generator.py

```diff
@@ -469,16 +474,18 @@ def _process_step(
                     new_sender = step.block_name
                 trackers.append(tracker.copy(new_sender))
 
-        new_trackers = []
+        end_trackers = []
```

Comment:
not sure about this naming, because the conversation is not finished at this point, is it?

Reply:
yes, but in `deduplication` the conversation didn't finish as well, but we call it `end_trackers` because we do not add any checkpointed story blocks to them. But if you have any suggestion, I'm open to change the name

Reply:
all good